tests/Unit/PrestaShopBundle/Utils/BoolParserTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Tests\Unit\PrestaShopBundle\Utils;
+
+use PHPUnit\Framework\TestCase;
+use PHPUnit_Framework_AssertionFailedError;
+use PrestaShopBundle\Utils\BoolParser;
+
+class BoolParserTest extends TestCase
+{
+    /**
+     * @throws PHPUnit_Framework_AssertionFailedError
+     */
+    public function testCastToBool()
+    {
+        $this->assertFalse(BoolParser::castToBool('false'));
```

Comment:
try casting an object or array :slightly_smiling_face:

Reply:
array currently not supported :)